### PR TITLE
Moving audit log content to security section

### PR DESCRIPTION
**Pull request by bergerhoffer**
Note to self: 
TODO: If this move is approved, add redirects.

Preview: 
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-view.html
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-policy-config.html

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@sttts @Anandnatraj I moved these audit log sections from "Nodes" to "Security", and removed references to it being "node audit logs". Can you please review and see if this looks better to you?

Preview: 
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-view.html
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-policy-config.html

**Comment by bergerhoffer** [comment]
@mburke5678 It was requested that we move the audit log sections from "Nodes" to "Security". This includes both the section on viewing audit logs that's been around since 4.1, and the new section I just added on configuring the audit log policy.

Do you/your Node team have any objections to this move? Thanks!

Preview:
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-view.html
* https://moving-audit-log-sections--ocpdocs.netlify.app/openshift-enterprise/latest/security/audit-log-policy-config.html

**Comment by Anandnatraj** [comment]
LGTM

**Comment by mburke5678** [comment]
@bergerhoffer Is there a reason to not have the information in Nodes *and* Security?

**Comment by bergerhoffer** [comment]
@mburke5678 That was my suggestion originally. I think that the section to configure the audit log policy definitely needs to be under security and not nodes. But I'm on the fence about the section for viewing audit logs. It's for the API server, but you seem to view it on a per-node basis.

@sttts Do you feel strongly that the section on viewing audit logs doesn't belong under Nodes, even though you use the `oc adm node-logs` command to view, and pass in a node (or role for the node)?

File: security/audit-log-policy-config.adoc [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+[id="audit-log-policy-config"]
+= Configuring the audit log policy
 include::modules/common-attributes.adoc[]
-:context: nodes-nodes-audit-config
+:context: audit-log-policy-config
 
 toc::[]
 
-You can control the amount of information that is logged to the node audit logs by choosing the audit log policy profile to use.
+You can control the amount of information that is logged to the audit logs by choosing the audit log policy profile to use.
```
Comment (sttts) [inline security/audit-log-policy-config.adoc:8]: would be more specific and call it "API server audit logs". It's not system audit.

**Comment by sttts** [comment]
SGTM

**Comment by sttts** [comment]
> Do you feel strongly that the section on viewing audit logs doesn't belong under Nodes, even though you use the oc adm node-logs command to view, and pass in a node (or role for the node)?

The files are accessed through the node. But the concept itself has nothing to do with nodes.

**Comment by bobfuru** [comment]
LGTM

**Comment by bergerhoffer** [comment]
Thanks all! This update will get merged as soon as the redirects are in place, hopefully this week. CC @vikram-redhat

**Comment by bergerhoffer** [comment]
Since the audit log config is new for 4.6, there will be conflicts when cherry picking back to 4.4 and 4.5. @vikram-redhat since you're going to be the one merging this, I'm updating this PR to be for 4.6 and 4.7, and opened #26854 for 4.4 and 4.5.

I'll update the redirects register to reflect these two PRs.

File: modules/nodes-nodes-audit-policy.adoc [hunk 2]
```diff
@@ -1,9 +1,9 @@
 // Module included in the following assemblies:
 //
-// * nodes/nodes/nodes-nodes-audit-config.adoc
+// * security/audit-log-policy-config.adoc
 
 [id="configuring-audit-policy_{context}"]
-= Configuring the node audit log policy
+= Configuring the audit log policy
```
Comment (mburke5678) [inline modules/nodes-nodes-audit-policy.adoc:6]: @bergerhoffer I see that you didn't add the sentence below. Would you be able to fix it up while you are in the file? it doesn't sound quite right to me.
_when logging requests that come_ -> _when logging the requests that come_ or _when logging requests come_

**Pull request merged**